src/bindgen/builder.rs

```diff
@@ -152,6 +154,12 @@ impl Builder {
         self
     }
 
+    #[allow(unused)]
```

Comment:
Why the `#[allow]`?

Reply:
Because every other method for that struct also has it. I didn't want to break that pattern :)